doc(ldap-docker): explain how to use ldap synchronizer docker image (#1610)
bonitasoft/bonita-doc: modules/ROOT/pages/docker-ldap-synchronizer.adoc

> = Deploy LDAP Synchronizer with Docker

Reviewer: This page is missing from the taxonomy.
Also, IMO, it would make more sense to have everything on the same page `ldap-synchronizer.adoc`

Author: ok

> = Deploy LDAP Synchronizer with Docker

Reviewer: This page should be remove now that it's in the other page

:description: How to install and use the LDAP Synchronizer docker  distribution.

How to install and use the LDAP Synchronizer docker distribution.

This guide assumes you have a working docker environment.

== Configuration

=== Environment Variables

All general connection settings( `bonita.properties`, `ldap.properties`

> and `log.properties`) are configurable with environment properties.

Reviewer: ```suggestion
and `loggger.properties`) are configurable with environment properties.
```

> Specific configuration available on `mapper.properties` and `sync.properties` could configurable with a copy of files in directory link by volume `/opt/conf/user-config`.

Reviewer: ```suggestion
`mapper.properties` and `sync.properties` can be configured by copying is in the volume `/opt/conf/user-config`. See below for an example on that.
```

Those files could be copied directly from ldap-syncrhonizer docker image with command to directory link to volume `/opt/conf/user-config` with  :

[source,bash,subs="verbatim,quotes"]
----
docker cp $(docker create --rm quay.io/bonitasoft/bonita-ldap-synchronizer:*version*):/conf/default/mapper.properties *directory_link_to_volume*
docker cp $(docker create --rm quay.io/bonitasoft/bonita-ldap-synchronizer:*version*):/conf/default/sync.properties *directory_link_to_volume*

> ----

Reviewer: Maybe here we can describe the steps to override the e.g. `sync.properties` configuration file? 
i.e.
* copy the file from the base image using the command above `docker cp [....]`.
* edit the file
* start the docker image with the volume `docker run [...] -v pwd:/opt/conf/user-config [..]`

Author: ok
